cmd/server-main.go

```diff
@@ -508,6 +508,15 @@ func serverMain(ctx *cli.Context) {
 		addrs = append(addrs, globalMinioAddr)
 	}
 
+	// Listen to local ipv4 and ipv6
```

Comment:
@vadmeste we should make sure to have addrs to be unique, deduplicating addrs is needed here. Otherwise multiple IPs to same bind port might fail.